streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -935,6 +944,14 @@ void shutdown(final boolean clean) {
         return tasksToCloseDirty;
     }
 
+    public void updateCurrentAssignmentTopology(final Set<String> assignedNamedTopologies) {
+        topologyMetadata.updateCurrentAssignmentTopology(assignedNamedTopologies);
```

Comment:
The propagated `assignedNamedTopologies` is not used yet anywhere in the KS code. Could you elaborate a bit how it would be used in the future?

Reply:
Removed until it's needed, if it is